docs/content/concepts/dynamic-fields/transfers/custom-rules.mdx

```diff
@@ -60,4 +60,4 @@ public fun transfer_locked(object: O) {
 }
 ```
 
-With these rules in place there are two different custom transfer rules for any object `O`; either it's unlocked and anyone can transfer it, or it's locked, and only `0xCAFE` can transfer it. Importantly, these two ways of transferring `O` are the only ways of transferring any object of type `O`. In particular, because `O` does not have the `store` ability, you cannot transfer it using the `sui::transfer::public_transfer` function. In fact, the only ways of transferring `O` are using `examples::custom_transfer::transfer_unlocked` and `examples::custom_transfer::transfer_locked`.
+With these rules in place, there are two different custom transfer rules for any object O: either it's unlocked and anyone can transfer it, or it's locked, and it can only be transferred to 0xCAFE. Importantly, these two ways of transferring `O` are the only ways of transferring any object of type `O`. In particular, because `O` does not have the `store` ability, you cannot transfer it using the `sui::transfer::public_transfer` function. In fact, the only ways of transferring `O` are using `examples::custom_transfer::transfer_unlocked` and `examples::custom_transfer::transfer_locked`.
```

Comment:
```suggestion
With these rules in place, there are two different custom transfer rules for any object `O`: either it's unlocked and anyone can transfer it, or it's locked, and it can only be transferred to `0xCAFE`. Importantly, these two ways of transferring `O` are the only ways of transferring any object of type `O`. In particular, because `O` does not have the `store` ability, you cannot transfer it using the `sui::transfer::public_transfer` function. In fact, the only ways of transferring `O` are using `examples::custom_transfer::transfer_unlocked` and `examples::custom_transfer::transfer_locked`.
```